lms/djangoapps/student_account/views.py

```diff
@@ -96,14 +96,24 @@ def login_and_registration_form(request, initial_mode="login"):
         except (KeyError, ValueError, IndexError):
             pass
 
+
```

Comment:
Did you see the one about translating a non literal string?  line 101 of lms/djangoapps/student_account/views.py